shared/js/views.jsx

```diff
@@ -701,8 +701,9 @@ loop.shared.views = (function(_, mozL10n) {
 
       var storeState = this.props.cursorStore.getStoreState();
 
-      var deltaX = event.clientX - storeState.videoLetterboxing.left;
-      var deltaY = event.clientY - storeState.videoLetterboxing.top;
+      var offset = this._getOffset(this.getDOMNode().querySelector("video"));
```

Comment:
Oh duh. We implicitly assumed the video was the top-left element, so that's why plain clientX/Y worked. We should be able to use `node.getBoundingClientRect()`, e.g.,

```
n = document.querySelector(".remote.focus-stream");
b = n.getBoundingClientRect();
n.addEventListener("mousemove", e => console.log(e.clientX - b.left, e.clientY - b.top));
```

Basically, it implements `_getOffset` for you.